inst/cpp_generation/model_input.csv

```diff
@@ -60,4 +60,7 @@ paed_art_age_spec,Children.hc_art_is_age_spec,int,ModelVariant::run_child_model,
 hc_age_coarse,Children.hc_age_coarse,real_type,ModelVariant::run_child_model,,FALSE,1,children.hcAG_end,,,
 abortion,Children.abortion,real_type,ModelVariant::run_child_model,,FALSE,2,children.hAB_ind,proj_years,,
 patients_reallocated,Children.patients_reallocated,real_type,ModelVariant::run_child_model,,FALSE,1,proj_years,,,
-hc_age_coarse_cd4,Children.hc_age_coarse_cd4,real_type,ModelVariant::run_child_model,,FALSE,1,15,,,
+hc_age_coarse_cd4,Children.hc_age_coarse_cd4,int,ModelVariant::run_child_model,,FALSE,1,options.p_idx_hiv_first_adult,,,
+inc,Children.adult_female_inf,real_type,ModelVariant::run_child_model,,FALSE,2,options.p_fertility_age_groups,proj_years,,
```

Comment:
what is `inc` / where does that label come from?  It might be better to have a more descriptive label here.

Reply:
That is the R label, I've changed it to match the C++ label for now. 

I also created a new issue where I will align all of the R and C++ names. 

https://github.com/mrc-ide/frogger/issues/103